src/routers/task.js

```diff
@@ -92,4 +94,47 @@ router.delete('/tasks/:id', auth, async(req, res) => {
     }
 })
 
+const upload = multer({
+    limits: {
+        fileSize: 1000000
+    },
+    fileFilter(req, file, callback) {
+        if (!file.originalname.match(/\.(jpg|jpeg|png)$/)) {
+            return callback(new Error('Please upload an image document.'))
+        }
+        callback(undefined, true)
+    }
+})
+
+router.post('/tasks/:id/logo', auth, upload.single('logo'), async(req, res) => {
+    try {
+        const task = await Task.findOne({ _id: req.params.id, createdBy: req.user._id })
+        if (!task) {
+            return res.status(404).send()
+        }
+        const buffer = await sharp(req.file.buffer).resize({ width: 250, height: 250 }).png().toBuffer()
```

Comment:
redundant variable

Reply:
Why is it redundant? It's used in the next line.

Reply:
the value of variable is used, but the variable is redundant it self

Reply:
I'm sorry, but how can a used value be redundant?

Reply:
Fixed.